config/management/src/validator_config.rs

```diff
@@ -187,3 +176,29 @@ pub fn get_owner_from_key(
     let owner_key = storage.ed25519_key(OWNER_KEY)?;
     Ok(account_address::from_public_key(&owner_key))
 }
+
+pub fn update_config_and_reconfigure(
```

Comment:
this seems like it should only be in the operational tool?

Reply:
Probably, I couldn't decide which location I wanted it in